Resources/Prototypes/Entities/Mobs/NPCs/animals.yml

```
@@ -3276,7 +3276,8 @@
   name: syndicat
   id: MobCatSyndy
   parent: MobCatSpace
-  description: Explosive kitten.
+  # description: Explosive kitten.
+  description: A cat in red, prepared to cause trouble. #DeltaV
```

More slight description changes.

```suggestion
  description: A adorable cat in a terrifying red suit, prepared to cause trouble. #DeltaV
```